dashboard/.sass-lint.yml

```diff
@@ -0,0 +1,43 @@
+# Linter Options
+options:
+  # Don't merge default rules
+  merge-default-rules: false
+  # Raise an error if more than 50 warnings are generated
+  max-warnings: 50
+# File Options
+files:
+  include: "src/**/*.s+(a|c)ss"
+  ignore:
+    - "node_modules/**/*.*"
+# Rule Configuration
+rules:
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+  mixins-before-declarations:
+    - 2
+    - exclude:
+        - breakpoint
+        - mq
+
+  no-warn: 1
+  no-debug: 1
+  no-ids: 2
+  no-important: 2
+  hex-notation:
+    - 2
+    - style: uppercase
+  indentation:
+    - 2
+    - size: 2
+  property-sort-order:
+    - 1
+    - order:
+        - display
+        - margin
+      ignore-custom-properties: true
+  variable-for-property:
+    - 1
```

Comment:
@Angelmmiguel this config is copied from the sass-lint sample (https://github.com/sasstools/sass-lint/blob/master/docs/sass-lint.yml), but I changed this from an error to a warning since we are using values for margin. What do you think about this config? Is there a preferred config we should be using? Should we disable this setting entirely?

Reply:
I think it's fine for now @prydonius. We don't have any official scss-lint config, so let's use this. I'll create a PR when we define the config in HEx :)